### DISCUSSION TITLE
Some Thoughts on Future Road Map

### DISCUSSION THREAD

**Discussion by edwinnglabs** [uber/orbit#656] [Ideas]
- Standard Bayesian linear regression model this covers
1. to provide a quick and simple way to perform analysis
2. to provide a feature selection properties such as Horseshoe prior

- Enhancement of summary of statistics on parameters such as 
1. upper and lower quantile and s.e
2. box plots of range of coefficients (we do have one in arviz but does it cover proper naming of regression coefficients etc. ?)
3. other standard analytics from BDA3 / statistical rethinking

- Start covering TFP and Numpyro Estimators

- Package usability 
1. move stan compilation in installation?
2. allow suppress message from pystan2 together with the `verbose` flag

thoughts @wangzhishi @GavinSteiningerUber @uber/orbit-dev ?

**Comment by wangzhishi** [comment]
![image](https://user-images.githubusercontent.com/7907663/148106675-ef223957-c11b-4190-bb35-73839ea86296.png)

**Comment by wangzhishi** [comment]
https://discourse.mc-stan.org/t/horseshoe-prior-in-rstan/14561
